aviary/subsystems/propulsion/utils.py

```diff
@@ -98,6 +104,65 @@ def convert_geopotential_altitude(altitude):
     return altitude
 
 
+class UncorrectData(om.Group):
+    def initialize(self):
+        self.options.declare(
+            'num_nodes', types=int, default=1)
+        self.options.declare(
+            'aviary_options', types=AviaryValues,
+            desc='collection of Aircraft/Mission specific options')
+
+    def setup(self):
+        num_nodes = self.options['num_nodes']
+
+        self.add_subsystem(
+            'pressure_term',
+            om.ExecComp(
+                'delta_T = (P0 * (1 + .2*mach**2)**3.5) / P_amb',
+                delta_T={'units': "unitless", 'shape': num_nodes},
+                P0={'units': 'psi', 'shape': num_nodes},
+                mach={'units': 'unitless', 'shape': num_nodes},
+                P_amb={'val': np.full(num_nodes, 14.696), 'units': 'psi', },
+                has_diag_partials=True,
+            ),
+            promotes_inputs=[
+                ('P0', Dynamic.Mission.STATIC_PRESSURE),
+                ('mach', Dynamic.Mission.MACH),
+            ],
+            promotes_outputs=['delta_T'],
+        )
+
+        self.add_subsystem(
+            'temperature_term',
+            om.ExecComp(
+                'theta_T = T0 * (1 + .2*mach**2)/T_amb',
+                theta_T={'units': "unitless", 'shape': num_nodes},
+                T0={'units': 'degR', 'shape': num_nodes},
+                mach={'units': 'unitless', 'shape': num_nodes},
+                T_amb={'val': np.full(num_nodes, 518.67), 'units': 'degR', },
+                has_diag_partials=True,
+            ),
+            promotes_inputs=[
+                ('T0', Dynamic.Mission.TEMPERATURE),
+                ('mach', Dynamic.Mission.MACH),
+            ],
+            promotes_outputs=['theta_T'],
+        )
+
+        self.add_subsystem(
+            'uncorrection',
+            om.ExecComp(
+                'data_uncorrected = data_corrected * (delta_T + theta_T**.5)',
+                shaft_power={'units': "hp", 'shape': num_nodes},
+                delta_T={'units': "unitless", 'shape': num_nodes},
+                theta_T={'units': "unitless", 'shape': num_nodes},
+                shaft_power_corrected={'units': "hp", 'shape': num_nodes},
```

Comment:
`shaft_power` and `shaft_power_corrected` need to be changed to `data`
I'd prefer `corrected_data` and `uncorrected_data`, since it is more natural than `data_corrected` and `data_uncorrected`. It will always be alias promoted to some actual variable, so I don't think we need to worry about keeping them together alphabetically.
Either way is fine

Reply:
Thanks for catching that, I'll update those variables. I'm fine with either order for the data variable names, so `XXX_data` is fine with me